static/styles/message.css

```diff
@@ -33,6 +33,25 @@ $first_column_width: 46px;
                 grid-column: 1 / 3;
```

Comment:
Nice catch!

Reply:
Hm, it looks like the timestamps and action icons are currently vertically centered rather than being at the top of the message, and I think that looks better than the screenshot above.

Reply:
<img width="664" alt="image" src="https://user-images.githubusercontent.com/25124304/203821257-d12bb719-5c8a-4908-a04f-f59dc80ec15f.png">

Thanks! Fixed!

Reply:
Thanks! I think the timestamp looks a bit too low in the editing UI for /me messages. Compare these two:

<img width="912" alt="Screen Shot 2022-11-24 at 10 54 02 PM" src="https://user-images.githubusercontent.com/2090066/203919188-42087297-6834-4d3c-9f72-878f2f423b21.png">

Once you're editing, I think it makes sense to align the timestamp at the top, as for other messages.

I checked on CZO, and we actually have the same issue there, so I'm fine with merging this PR and fixing it separately if that works best.

Reply:
Fixed
<img width="647" alt="image" src="https://user-images.githubusercontent.com/25124304/204183494-d9dafb62-5e9f-4775-97ea-526129afa7fc.png">

Reply:
Thanks! That screenshot looks good to me.